ee/src/main/java/org/jboss/as/ee/component/deployers/ModuleJndiBindingProcessor.java

```diff
@@ -86,6 +87,7 @@
 public class ModuleJndiBindingProcessor implements DeploymentUnitProcessor {
 
     private final boolean appclient;
+    private static final boolean IGNORE_UNUSED_RESOURCE_BINDING = Boolean.valueOf(WildFlySecurityManager.getPropertyPrivileged("jboss.ee.ignore-unused-resource-binding", "false"));
```

Comment:
Not required but possibly useful would be to make this an instance var, not static, so we (e.g. in a test) or users (for ??? reason) can manipulate this behavior using a server reload.